.gitignore

```diff
@@ -37,8 +37,6 @@ lerna-debug.log
 /public/
 
 /server/devdocs/search-index.js
-/server/devdocs/components-usage-stats.json
-/server/devdocs/proptypes-index.json
```

Comment:
Are these likely to be sitting around on devs' working copies? There's a risk of them accidentally being committed, nothing will force their removal.

Ideas for prevention:
- Temporarily leave them ignored.
- Add removal to the most common scripts (pre-commit?) for a period of time.
- Post about the change for awareness.

Reply:
Good point about the ignore files: I'm choosing solution No1, to keep ignoring them, and reverted changes to the `*ignore` files.